socialhome/streams/templatetags/slurp_json_context.py

```diff
@@ -0,0 +1,36 @@
+from django.utils.safestring import mark_safe
+from django.template import Library
+import json
+
+register = Library()
+
+
+@register.simple_tag(takes_context=True)
+def slurp_json_context(context):
+    """
+    Serializes ``context["json_context"]`` into JSON and generates a JS
+    script to attach it to ``window.context``
+    Args:
+        context: Current view context
+
+    Returns: "<script>window.context = {<context["json_context"]>}</script>"
+
+    """
+    return _slurp_json_context(context, lambda obj: str(obj))
+
+
+@register.simple_tag(takes_context=True)
+def slurp_json_context_unsafe(context):
```

Comment:
No, it is. I used it for debugging puposes. When I ported the code from the template I forgot to add parenthesis to seme function calls and had a nasty string representation in my JSON code because of `lambda obj: str(obj)`.